### PR TITLE
Update changelog templates to use new labels

### PR DESCRIPTION
**Pull request by coderobe**
See-also: https://github.com/paritytech/labels/pull/2

### CODE REVIEW

**Comment by coderobe** [comment]
The C labels should work as-is from what i can tell

**Review by chevdor — APPROVED** [review]
Looking good overall.

No discussions about the trivial switches from one label to another.

Did you carefully test/think about the reduction of the logic you did ?

Like from 
```
if pr.meta.B and pr.meta.B.value != 5 and pr.meta.B.value != 7 or pr.meta.C or not pr.meta.B
```
to 
```
if pr.meta.T and pr.meta.T.value == 0
```

That seems much more open. It may not be an issue if the new rules (cf ruled_labels) ensure those cases you removed cannot go through.

**Review by EgorPopelyaev — APPROVED** [review]

**Comment by coderobe** [comment]
We'll see if any issues come up, we can adjust from there - i'd say we should merge this sooner rather than later and iron out issues afterwards, so that joyce & co can continue with the new labels.

**Comment by coderobe** [comment]
bot merge

File: scripts/ci/changelog/templates/change.md.tera [hunk 1]
```diff
@@ -33,10 +33,5 @@
 {%- set repo = "   " -%}
 {%- endif -%}
 
-{%- if c.meta.T and c.meta.T.value == 6 -%}
-{%- set xcm = " [✉️ XCM]" -%}
-{%- else -%}
-{%- set xcm = "" -%}
-{%- endif -%}
```
Comment (the-right-joyce) [inline scripts/ci/changelog/templates/change.md.tera:40]: we still have the XCM tag, which is T6-XCM
So if you want to continue using it in the release notes, it's provided

Comment (coderobe) [reply scripts/ci/changelog/templates/change.md.tera:40]: ah good catch

**Pull request merged**